tests/proxysql-admin-testsuite.bats

```diff
@@ -121,13 +121,24 @@ fi
 }
 
 
+@test "run proxysql-admin --update-mysql-version ($WSREP_CLUSTER_NAME)" {
```

Comment:
It seems like this can be merged into the test below.

Reply:
Merged